proxy/delete.go

```diff
@@ -5,51 +5,42 @@ package proxy
 
 import (
 	"bytes"
+	"context"
 	"encoding/json"
 	"fmt"
 	"io/ioutil"
 	"net/http"
-	"strings"
 
 	"github.com/openfaas/faas/gateway/requests"
 )
 
-// DeleteFunction delete a function from the OpenFaaS server
-func DeleteFunction(gateway string, functionName string, tlsInsecure bool, namespace string) error {
-	return DeleteFunctionToken(gateway, functionName, tlsInsecure, "", namespace)
+type NamespaceParam struct {
```

Comment:
Public structs need documentation

Reply:
Remove this struct as it is not required.

Reply:
Was this done?

Reply:
yes .. I removed this as this is not required because now we are using `net.URL`.